gateway/bin/apicast

```diff
@@ -129,7 +129,7 @@ local function resty(code)
 
     if not cmd then return nil, 'could not find resty' end
 
-    local handle = io.popen(([[/usr/bin/env -i %q -e %q]]):format(cmd, code))
+    local handle = io.popen(([[/usr/bin/env -i %q -e %q 2>/dev/null]]):format(cmd, code))
```

Comment:
This code change looks unrelated to this issue. I suspect it's unintended to be included in this commit and will cause a conflict with other PRs or with this one depending which gets merged first.